tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
IMO, if `@A - @B` means "rules from A that are not in B", then having an explicitly disabled rule in B is like not having it at all. I would expect `['foo' => true] - ['foo' => false]` to result in `['foo' => true]`.